python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -924,6 +936,15 @@ def _getattribute(self, name):
 
     return type(name, (GPIProxyObject,), d)
 
+_metaClassHolder = None
+def _getMetaClass():
+    global _metaClassHolder
+    if _metaClassHolder is None:
+        from Ganga.GPIDev.Base.Objects import ObjectMetaclass
+        _metaClassHolder = ObjectMetaclass
+    return _metaClassHolder
```

Comment:
That sounds like a good idea. If the proxy layer is doing its job properly (which we both know it doesn't always) then we should be able to remove all usage of `stripProxy` outside of Proxy.py (and maybe one or two other proxy-layer classes). Simplifying this logic inside `GangaObject` will only help us in the long-run as we look towards replacing it entirely.